README.md

```diff
@@ -0,0 +1,43 @@
+# Babelfish Tools
+
+[![Build Status](https://travis-ci.org/bblfsh/tools.svg?branch=master)](https://travis-ci.org/bblfsh/tools)
+[![codecov](https://codecov.io/gh/bblfsh/tools/branch/master/graph/badge.svg)](https://codecov.io/gh/bblfsh/tools)
+
+Language analysis tools on top of Babelfish
+
+## Build
+
+### With docker
+
+`make build`
+
+### Without docker
+
+`make build-internal`
+
+## Usage
+
+Babelfish Tools provides a set of tools built on top of Babelfish, to
+see which tools are supported, run:
+
+`bblfsh-tools --help`
+
+To make use of any of these tools you need to have the Babelfish
+server up and running. Look at
+[server site](https://github.com/bblfsh/server/) for details.
```

Comment:
It'd be probably better to add a oneliner aswell with a command to get the server image and run it. What will be the proper way to get the server image?